### ISSUE TITLE
Support for karma preprocessors 

### ISSUE THREAD

**Issue by nicojs** [stryker-mutator/stryker-karma-runner#19]
In order to support things like karma preprocessors we should run child processes from their respective sandbox folder. I tested stryker with a karma preprocessor here: https://github.com/nicojs/karma-preprocessor-stryker-test and it doesn't work.

**Comment by simondel** [comment]
Fixed with #19. When do you want to release this @nicojs?

**Comment by nicojs** [comment]
Yes please :)

**Comment by nicojs** [comment]
@zxxc we just released 0.3.5 of stryker-karma-runner, can you confirms this fixes the issue for you?

**Comment by chrabia** [comment]
Hello,
could you please explain how to set up karma and preprocessors in grunt file (without using stryker.conf.js)? Some simple example would be nice.

**Comment by zxxc** [comment]
@nicojs yes, angular template preprocessor is working from default karma.conf.js
seems we can close this issue
thanks for your help

**Comment by xxxtonixxx** [comment]
I have the same propblem when I try to use the @angular/cli preproccessor

**Comment by tgold007596** [comment]
We have a project running Closure framework, we added Jasmine for testing and Karma as a test runner, now we are trying to integrate Stryker for mutation testing, we are facing difficulties integrating it, it cannot read and locate the tests,
Stryker has open issues since 2017 that it doesn't execute Karma preprocessors.
In our project, Karma cannot run without preprocessors, because the project depends on google-closure-library.
I tried to update the Karma version from 0.1 to 5.0 but the issue still happens.
I also tried to execute Karma with the compiled files but it didn't work either
 
If you can help me and my team to solve this issue I will appreciate it very much.
 
Thanks,
Tal Goldberg
